src/core/library.c

```diff
@@ -413,6 +458,9 @@ MsQuicLibraryInitialize(
     MsQuicLib.ToeplitzHash.InputSize = CXPLAT_TOEPLITZ_INPUT_SIZE_QUIC;
     CxPlatToeplitzHashInitialize(&MsQuicLib.ToeplitzHash);
 
+    CxPlatDispatchRwLockInitialize(&MsQuicLib.StatelessRetryLock);
+    RetryLockInitialized = TRUE;
```

Comment:
There is no failure path between this and `PlatformInitialized = TRUE;`. Let's instead just use `PlatformInitialized` and set it here.